tests/components/wiser/test_config_flow.py

```diff
@@ -0,0 +1,90 @@
+"""Test the Wiser Heating Component for Home Assistant config flow."""
+from homeassistant import config_entries, setup
+from homeassistant.components.wiser.config_flow import CannotConnect, InvalidAuth
+from homeassistant.components.wiser.const import DOMAIN
+
+from tests.async_mock import patch
+
+
+async def test_form(hass):
+    """Test we get the form."""
+    await setup.async_setup_component(hass, "persistent_notification", {})
+    result = await hass.config_entries.flow.async_init(
+        DOMAIN, context={"source": config_entries.SOURCE_USER}
+    )
+    assert result["type"] == "form"
+    assert result["errors"] == {}
+
+    with patch(
+        "homeassistant.components.wiser.config_flow.PlaceholderHub.authenticate",
+        return_value=True,
+    ), patch(
+        "homeassistant.components.wiser.async_setup", return_value=True
+    ) as mock_setup, patch(
+        "homeassistant.components.wiser.async_setup_entry", return_value=True,
+    ) as mock_setup_entry:
+        result2 = await hass.config_entries.flow.async_configure(
+            result["flow_id"],
+            {
+                "host": "1.1.1.1",
+                "username": "test-username",
+                "password": "test-password",
+            },
```

Comment:
Seems like this dictionary is reused quite a few times. Maybe put it in a variable to it can be reused?

Reply:
done and fixed tests